src/docs/Components/Form/TextInput/index.mdx

```diff
@@ -4,13 +4,15 @@ order: 11
 description: 'Inputs are used to create input fields with text on a single line. Their states depends on the user interaction or the context'
 links:
   vue:
-    link: 'https://adeo.github.io/mozaic-vue/?path=/story/components-mtextinput--try-it'
+    link: 'https://adeo.github.io/mozaic-vue/?path=/story/components-mtextinput'
```

Comment:
The link is broken.
The good url is => https://adeo.github.io/mozaic-vue/?path=/story/components-form-mtextinput--default